https://issues.redhat.com/browse/ACM-23489 Secure clusters tech debt updates (#8239)
open-cluster-management/rhacm-docs: secure_clusters/certificates.adoc

[#acm-hub-cluster-certs]
== {acm-short} hub cluster certificates

> {ocp-short} default ingress certificate is a type of hub cluster certificate. After the {acm-short} installation, Observability certificates are created and used by the Observability components to give mutual TLS on the traffic between the hub cluster and managed cluster. Access the Observability namespaces to retrieve and implement the different Observability certificates, depending on your needs.

Reviewer: little long of a sentence. Disregard my comments if my changes are technically inaccurate or looses some of the necessary information to the user: 

```suggestion
{ocp-short} default ingress certificate is a type of hub cluster certificate. After the {acm-short} installation,  Observability components use Observability certificates to give mutual TLS to the hub and managed clusters. Access the Observability namespaces to retrieve and implement the different Observability certificates.
```

> {ocp-short} default ingress certificate is a type of hub cluster certificate. After the {acm-short} installation, Observability certificates are created and used by the Observability components to give mutual TLS on the traffic between the hub cluster and managed cluster. Access the Observability namespaces to retrieve and implement the different Observability certificates, depending on your needs.

Author: ```suggestion
{ocp-short} default ingress certificate is a type of hub cluster certificate. After you install {acm-short}, Observability certificates are created and used by the Observability components to give mutual TLS for traffic between the hub and managed cluster. Access the Observability namespaces to retrieve and implement the different Observability certificates.
```

| open-cluster-management-agent-addon
| cluster-proxy-service-proxy-server-certificates
|=== 

[#additional-resources-cert]
== Additional resources

- Use the certificate policy controller to create and manage certificate policies on managed clusters. See link:../governance/cert_policy_ctrl.adoc#certificate-policy-controller[Certificate policy controller] for more details.

> - for more details about securely connecting to a privately-hosted Git server with SSL/TLS certificates, see link:../applications/configuring_git_channel.adoc#using-custom-CA-certificates-for-secure-HTTPS-connection[Using custom CA certificates for a secure HTTPS connection].

Author: ```suggestion
- For more details about securely connecting to a privately-hosted Git server with SSL/TLS certificates, see link:../applications/configuring_git_channel.adoc#using-custom-CA-certificates-for-secure-HTTPS-connection[Using custom CA certificates for a secure HTTPS connection].
```
